apis/datadoghq/v2alpha1/zz_generated.openapi.go

```diff
@@ -19,33 +19,33 @@ import (
 
 func GetOpenAPIDefinitions(ref common.ReferenceCallback) map[string]common.OpenAPIDefinition {
 	return map[string]common.OpenAPIDefinition{
-		"./apis/datadoghq/v2alpha1.CSPMHostBenchmarksConfig":          schema__apis_datadoghq_v2alpha1_CSPMHostBenchmarksConfig(ref),
```

Comment:
Could you check this please as well, diff shouldn't be this big. It also causes CI failure.

Reply:
This file is automatically generated by running `make generate manifests` 

For some reason, if $GOPATH is set, it results in a huge diff. 

Setting $GOPATH to "/tmp" temporarily fixes the issue. 

I am not sure why this is the case.